report-api/report-templates/payment_receipt.html

```diff
@@ -419,19 +419,19 @@
               BC Online Account Number: <span> {{bcOnlineAccountNumber}}</span>
             </div>
             {% endif %}
-            {% if invoice.get('folioNumber', 'None') != 'None' %}
+            {% if invoice.get('folioNumber', None) %}
             <div class="payment-row">
               Folio Number: <span> {{invoice['folioNumber']}}</span>
             </div>
             {% endif %}
-            {% if routingSlipNumber %}
+            {% if invoice.get('routingSlip', None) %}
```

Comment:
i forgot .just making sure... does this change need no change in pay-api?

Reply:
Pay already passes this, as it passes the whole invoice dict to report-api.